jenkins/sars_cov_2.py

```diff
@@ -32,7 +32,7 @@ def get_expected_output_files(output_path: str, sample_ids: List[str], sequencin
         sequencing_technology=sequencing_technology,
     )
     # generate a unified list of paths as non-sample results files must also be included
-    output_files = [path for sample_paths in output_files_per_sample.values() for path in sample_paths]
+    output_files = [f["file"] for sample_files in output_files_per_sample.values() for f in sample_files]
```

Comment:
in the jenkins integration test, we only check the existence of the output files, not the file type. The logic for the file type is tested as part of the unit tests.